packages/env/lib/commands/run.js

```diff
@@ -61,16 +61,22 @@ function spawnCommandDirectly( config, container, command, envCwd, spinner ) {
 	const hostUser = getHostUser();
 
 	// We need to pass absolute paths to the container.
-	envCwd = path.resolve( '/var/www/html', envCwd );
+	envCwd = path.resolve(
+		container === 'composer' ? '/app' : '/var/www/html',
+		envCwd
+	);
 
+	const useRun = container === 'phpunit' || container === 'composer';
 	const isTTY = process.stdout.isTTY;
+
 	const composeCommand = [
 		'-f',
 		config.dockerComposeConfigPath,
-		'exec',
+		useRun ? 'run' : 'exec',
 		! isTTY ? '--no-TTY' : '',
 		'-w',
 		envCwd,
+		useRun ? '--rm' : '',
```

Comment:
I tested that empty string is acceptable when running exec (`npx wp-env run cli ls`)